CHANGELOG.md

```diff
@@ -24,9 +24,8 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 - [#2208](https://github.com/thanos-io/thanos/pull/2208) Query and Rule: fix handling of `web.route-prefix` to correctly handle `/` and prefixes that do not begin with a `/`.
 - [#2311](https://github.com/thanos-io/thanos/pull/2311) Receive: ensure receive component serves TLS when TLS configuration is provided.
 - [#2319](https://github.com/thanos-io/thanos/pull/2319) Query: fixed inconsistent naming of metrics.
-- [#2390](https://github.com/thanos-io/thanos/pull/2390) Store: Fixed bug which was causing all posting offsets to be used instead of 1/32 as it was meant.
-Added hidden flag to control this behavior.
-- [#2393](https://github.com/thanos-io/thanos/pull/2393) Store: Fixed bug causing certain not-existing label values queried to fail with "invalid-size" error from binary header.
+- [#2390](https://github.com/thanos-io/thanos/pull/2390) Store: fixed bug that was causing all posting offsets to be used instead of only 1/32 as intended; added hidden flag to control this behavior.
```

Comment:
We are missing those:

https://github.com/thanos-io/thanos/commit/e1b1d716adfc838f776036ba132090c7644bed9e
https://github.com/thanos-io/thanos/commit/c05daeeace272a94fae81fd0c15ff70adab50e10